[2019-02-23 11:14:29.825] [INFO] console - Express server listening on :8999, in development mode
[2019-02-23 11:14:29.851] [INFO] console - mongo连接成功
[2019-02-23 11:14:29.854] [INFO] console - mongo连接打开
[2019-02-23 11:14:52.077] [ERROR] console - (node:47891) DeprecationWarning: Mongoose: mpromise (mongoose's default promise library) is deprecated, plug in your own promise library instead: http://mongoosejs.com/docs/promises.html
[2019-02-23 11:20:11.034] [INFO] console - Express server listening on :8999, in development mode
[2019-02-23 11:20:11.058] [INFO] console - mongo连接成功
[2019-02-23 11:20:11.060] [INFO] console - mongo连接打开
[2019-02-23 11:20:16.910] [ERROR] console - (node:48641) DeprecationWarning: Mongoose: mpromise (mongoose's default promise library) is deprecated, plug in your own promise library instead: http://mongoosejs.com/docs/promises.html
[2019-02-23 11:20:24.266] [INFO] console - ******** getResFn error ******** 
[2019-02-23 11:20:24.269] [INFO] console - { 113: duplicated username
    at fail (/Users/rogersmac/ML/authCenter/utils/controller/index.js:47:19)
    at /Users/rogersmac/ML/authCenter/apps/auth/app.js:40:13
    at Generator.next (<anonymous>)
    at onFulfilled (/Users/rogersmac/ML/authCenter/node_modules/co/index.js:65:19)
    at newTickHandler (/Users/rogersmac/ML/authCenter/node_modules/mpromise/lib/promise.js:234:18)
    at _combinedTickCallback (internal/process/next_tick.js:131:7)
    at process._tickCallback (internal/process/next_tick.js:180:9) name: 113 }
[2019-02-23 11:20:24.277] [ERROR] console - { 113: duplicated username
    at fail (/Users/rogersmac/ML/authCenter/utils/controller/index.js:47:19)
    at /Users/rogersmac/ML/authCenter/apps/auth/app.js:40:13
    at Generator.next (<anonymous>)
    at onFulfilled (/Users/rogersmac/ML/authCenter/node_modules/co/index.js:65:19)
    at newTickHandler (/Users/rogersmac/ML/authCenter/node_modules/mpromise/lib/promise.js:234:18)
    at _combinedTickCallback (internal/process/next_tick.js:131:7)
    at process._tickCallback (internal/process/next_tick.js:180:9) name: 113 } '113: duplicated username\n    at fail (/Users/rogersmac/ML/authCenter/utils/controller/index.js:47:19)\n    at /Users/rogersmac/ML/authCenter/apps/auth/app.js:40:13\n    at Generator.next (<anonymous>)\n    at onFulfilled (/Users/rogersmac/ML/authCenter/node_modules/co/index.js:65:19)\n    at newTickHandler (/Users/rogersmac/ML/authCenter/node_modules/mpromise/lib/promise.js:234:18)\n    at _combinedTickCallback (internal/process/next_tick.js:131:7)\n    at process._tickCallback (internal/process/next_tick.js:180:9)'
[2019-02-23 11:22:43.713] [INFO] console - Express server listening on :8999, in development mode
[2019-02-23 11:22:43.748] [INFO] console - mongo连接成功
[2019-02-23 11:22:43.751] [INFO] console - mongo连接打开
[2019-02-23 11:22:49.132] [ERROR] console - (node:48961) DeprecationWarning: Mongoose: mpromise (mongoose's default promise library) is deprecated, plug in your own promise library instead: http://mongoosejs.com/docs/promises.html
[2019-02-23 11:33:33.184] [INFO] console - Express server listening on :8999, in development mode
[2019-02-23 11:33:33.225] [INFO] console - mongo连接成功
[2019-02-23 11:33:33.227] [INFO] console - mongo连接打开
[2019-02-23 11:33:47.398] [INFO] console - Express server listening on :8999, in development mode
[2019-02-23 11:33:47.423] [INFO] console - mongo连接成功
[2019-02-23 11:33:47.426] [INFO] console - mongo连接打开
[2019-02-23 11:33:51.202] [ERROR] console - (node:50402) DeprecationWarning: Mongoose: mpromise (mongoose's default promise library) is deprecated, plug in your own promise library instead: http://mongoosejs.com/docs/promises.html
[2019-02-23 11:33:51.209] [INFO] console - ******** getResFn error ******** 
[2019-02-23 11:33:51.211] [INFO] console - Error: invalid expiresIn option for string payload
    at Object.module.exports [as sign] (/Users/rogersmac/ML/authCenter/node_modules/jsonwebtoken/sign.js:128:22)
    at /Users/rogersmac/ML/authCenter/apps/auth/app.js:18:33
    at Generator.next (<anonymous>)
    at onFulfilled (/Users/rogersmac/ML/authCenter/node_modules/co/index.js:65:19)
    at newTickHandler (/Users/rogersmac/ML/authCenter/node_modules/mpromise/lib/promise.js:234:18)
    at _combinedTickCallback (internal/process/next_tick.js:131:7)
    at process._tickCallback (internal/process/next_tick.js:180:9)
[2019-02-23 11:33:51.219] [ERROR] console - Error: invalid expiresIn option for string payload
    at Object.module.exports [as sign] (/Users/rogersmac/ML/authCenter/node_modules/jsonwebtoken/sign.js:128:22)
    at /Users/rogersmac/ML/authCenter/apps/auth/app.js:18:33
    at Generator.next (<anonymous>)
    at onFulfilled (/Users/rogersmac/ML/authCenter/node_modules/co/index.js:65:19)
    at newTickHandler (/Users/rogersmac/ML/authCenter/node_modules/mpromise/lib/promise.js:234:18)
    at _combinedTickCallback (internal/process/next_tick.js:131:7)
    at process._tickCallback (internal/process/next_tick.js:180:9) 'Error: invalid expiresIn option for string payload\n    at Object.module.exports [as sign] (/Users/rogersmac/ML/authCenter/node_modules/jsonwebtoken/sign.js:128:22)\n    at /Users/rogersmac/ML/authCenter/apps/auth/app.js:18:33\n    at Generator.next (<anonymous>)\n    at onFulfilled (/Users/rogersmac/ML/authCenter/node_modules/co/index.js:65:19)\n    at newTickHandler (/Users/rogersmac/ML/authCenter/node_modules/mpromise/lib/promise.js:234:18)\n    at _combinedTickCallback (internal/process/next_tick.js:131:7)\n    at process._tickCallback (internal/process/next_tick.js:180:9)'
[2019-02-23 11:34:39.026] [INFO] console - ******** getResFn error ******** 
[2019-02-23 11:34:39.028] [INFO] console - Error: invalid expiresIn option for string payload
    at Object.module.exports [as sign] (/Users/rogersmac/ML/authCenter/node_modules/jsonwebtoken/sign.js:128:22)
    at /Users/rogersmac/ML/authCenter/apps/auth/app.js:18:33
    at Generator.next (<anonymous>)
    at onFulfilled (/Users/rogersmac/ML/authCenter/node_modules/co/index.js:65:19)
    at newTickHandler (/Users/rogersmac/ML/authCenter/node_modules/mpromise/lib/promise.js:234:18)
    at _combinedTickCallback (internal/process/next_tick.js:131:7)
    at process._tickCallback (internal/process/next_tick.js:180:9)
[2019-02-23 11:34:39.033] [ERROR] console - Error: invalid expiresIn option for string payload
    at Object.module.exports [as sign] (/Users/rogersmac/ML/authCenter/node_modules/jsonwebtoken/sign.js:128:22)
    at /Users/rogersmac/ML/authCenter/apps/auth/app.js:18:33
    at Generator.next (<anonymous>)
    at onFulfilled (/Users/rogersmac/ML/authCenter/node_modules/co/index.js:65:19)
    at newTickHandler (/Users/rogersmac/ML/authCenter/node_modules/mpromise/lib/promise.js:234:18)
    at _combinedTickCallback (internal/process/next_tick.js:131:7)
    at process._tickCallback (internal/process/next_tick.js:180:9) 'Error: invalid expiresIn option for string payload\n    at Object.module.exports [as sign] (/Users/rogersmac/ML/authCenter/node_modules/jsonwebtoken/sign.js:128:22)\n    at /Users/rogersmac/ML/authCenter/apps/auth/app.js:18:33\n    at Generator.next (<anonymous>)\n    at onFulfilled (/Users/rogersmac/ML/authCenter/node_modules/co/index.js:65:19)\n    at newTickHandler (/Users/rogersmac/ML/authCenter/node_modules/mpromise/lib/promise.js:234:18)\n    at _combinedTickCallback (internal/process/next_tick.js:131:7)\n    at process._tickCallback (internal/process/next_tick.js:180:9)'
[2019-02-23 11:34:55.647] [INFO] console - Express server listening on :8999, in development mode
[2019-02-23 11:34:55.676] [INFO] console - mongo连接成功
[2019-02-23 11:34:55.679] [INFO] console - mongo连接打开
[2019-02-23 11:35:00.622] [ERROR] console - (node:50556) DeprecationWarning: Mongoose: mpromise (mongoose's default promise library) is deprecated, plug in your own promise library instead: http://mongoosejs.com/docs/promises.html
[2019-02-23 11:36:02.842] [INFO] console - ******** getResFn error ******** 
[2019-02-23 11:36:02.845] [INFO] console - { 401: authorization failed
    at fail (/Users/rogersmac/ML/authCenter/utils/controller/index.js:47:19)
    at /Users/rogersmac/ML/authCenter/apps/auth/app.js:62:13
    at Generator.next (<anonymous>)
    at onFulfilled (/Users/rogersmac/ML/authCenter/node_modules/co/index.js:65:19)
    at /Users/rogersmac/ML/authCenter/node_modules/co/index.js:54:5
    at new Promise (<anonymous>)
    at co (/Users/rogersmac/ML/authCenter/node_modules/co/index.js:50:10)
    at toPromise (/Users/rogersmac/ML/authCenter/node_modules/co/index.js:118:63)
    at next (/Users/rogersmac/ML/authCenter/node_modules/co/index.js:99:29)
    at onFulfilled (/Users/rogersmac/ML/authCenter/node_modules/co/index.js:69:7)
    at /Users/rogersmac/ML/authCenter/node_modules/co/index.js:54:5
    at new Promise (<anonymous>)
    at co (/Users/rogersmac/ML/authCenter/node_modules/co/index.js:50:10)
    at /Users/rogersmac/ML/authCenter/utils/controller/index.js:6:9
    at Layer.handle [as handle_request] (/Users/rogersmac/ML/authCenter/node_modules/express/lib/router/layer.js:95:5)
    at next (/Users/rogersmac/ML/authCenter/node_modules/express/lib/router/route.js:137:13)
    at Route.dispatch (/Users/rogersmac/ML/authCenter/node_modules/express/lib/router/route.js:112:3)
    at Layer.handle [as handle_request] (/Users/rogersmac/ML/authCenter/node_modules/express/lib/router/layer.js:95:5)
    at /Users/rogersmac/ML/authCenter/node_modules/express/lib/router/index.js:281:22
    at Function.process_params (/Users/rogersmac/ML/authCenter/node_modules/express/lib/router/index.js:335:12)
    at next (/Users/rogersmac/ML/authCenter/node_modules/express/lib/router/index.js:275:10)
    at Function.handle (/Users/rogersmac/ML/authCenter/node_modules/express/lib/router/index.js:174:3) name: 401 }
[2019-02-23 11:36:02.853] [ERROR] console - { 401: authorization failed
    at fail (/Users/rogersmac/ML/authCenter/utils/controller/index.js:47:19)
    at /Users/rogersmac/ML/authCenter/apps/auth/app.js:62:13
    at Generator.next (<anonymous>)
    at onFulfilled (/Users/rogersmac/ML/authCenter/node_modules/co/index.js:65:19)
    at /Users/rogersmac/ML/authCenter/node_modules/co/index.js:54:5
    at new Promise (<anonymous>)
    at co (/Users/rogersmac/ML/authCenter/node_modules/co/index.js:50:10)
    at toPromise (/Users/rogersmac/ML/authCenter/node_modules/co/index.js:118:63)
    at next (/Users/rogersmac/ML/authCenter/node_modules/co/index.js:99:29)
    at onFulfilled (/Users/rogersmac/ML/authCenter/node_modules/co/index.js:69:7)
    at /Users/rogersmac/ML/authCenter/node_modules/co/index.js:54:5
    at new Promise (<anonymous>)
    at co (/Users/rogersmac/ML/authCenter/node_modules/co/index.js:50:10)
    at /Users/rogersmac/ML/authCenter/utils/controller/index.js:6:9
    at Layer.handle [as handle_request] (/Users/rogersmac/ML/authCenter/node_modules/express/lib/router/layer.js:95:5)
    at next (/Users/rogersmac/ML/authCenter/node_modules/express/lib/router/route.js:137:13)
    at Route.dispatch (/Users/rogersmac/ML/authCenter/node_modules/express/lib/router/route.js:112:3)
    at Layer.handle [as handle_request] (/Users/rogersmac/ML/authCenter/node_modules/express/lib/router/layer.js:95:5)
    at /Users/rogersmac/ML/authCenter/node_modules/express/lib/router/index.js:281:22
    at Function.process_params (/Users/rogersmac/ML/authCenter/node_modules/express/lib/router/index.js:335:12)
    at next (/Users/rogersmac/ML/authCenter/node_modules/express/lib/router/index.js:275:10)
    at Function.handle (/Users/rogersmac/ML/authCenter/node_modules/express/lib/router/index.js:174:3) name: 401 } '401: authorization failed\n    at fail (/Users/rogersmac/ML/authCenter/utils/controller/index.js:47:19)\n    at /Users/rogersmac/ML/authCenter/apps/auth/app.js:62:13\n    at Generator.next (<anonymous>)\n    at onFulfilled (/Users/rogersmac/ML/authCenter/node_modules/co/index.js:65:19)\n    at /Users/rogersmac/ML/authCenter/node_modules/co/index.js:54:5\n    at new Promise (<anonymous>)\n    at co (/Users/rogersmac/ML/authCenter/node_modules/co/index.js:50:10)\n    at toPromise (/Users/rogersmac/ML/authCenter/node_modules/co/index.js:118:63)\n    at next (/Users/rogersmac/ML/authCenter/node_modules/co/index.js:99:29)\n    at onFulfilled (/Users/rogersmac/ML/authCenter/node_modules/co/index.js:69:7)\n    at /Users/rogersmac/ML/authCenter/node_modules/co/index.js:54:5\n    at new Promise (<anonymous>)\n    at co (/Users/rogersmac/ML/authCenter/node_modules/co/index.js:50:10)\n    at /Users/rogersmac/ML/authCenter/utils/controller/index.js:6:9\n    at Layer.handle [as handle_request] (/Users/rogersmac/ML/authCenter/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/rogersmac/ML/authCenter/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/Users/rogersmac/ML/authCenter/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/Users/rogersmac/ML/authCenter/node_modules/express/lib/router/layer.js:95:5)\n    at /Users/rogersmac/ML/authCenter/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/Users/rogersmac/ML/authCenter/node_modules/express/lib/router/index.js:335:12)\n    at next (/Users/rogersmac/ML/authCenter/node_modules/express/lib/router/index.js:275:10)\n    at Function.handle (/Users/rogersmac/ML/authCenter/node_modules/express/lib/router/index.js:174:3)'
[2019-02-23 11:36:04.974] [INFO] console - ******** getResFn error ******** 
[2019-02-23 11:36:04.977] [INFO] console - { 401: authorization failed
    at fail (/Users/rogersmac/ML/authCenter/utils/controller/index.js:47:19)
    at /Users/rogersmac/ML/authCenter/apps/auth/app.js:62:13
    at Generator.next (<anonymous>)
    at onFulfilled (/Users/rogersmac/ML/authCenter/node_modules/co/index.js:65:19)
    at /Users/rogersmac/ML/authCenter/node_modules/co/index.js:54:5
    at new Promise (<anonymous>)
    at co (/Users/rogersmac/ML/authCenter/node_modules/co/index.js:50:10)
    at toPromise (/Users/rogersmac/ML/authCenter/node_modules/co/index.js:118:63)
    at next (/Users/rogersmac/ML/authCenter/node_modules/co/index.js:99:29)
    at onFulfilled (/Users/rogersmac/ML/authCenter/node_modules/co/index.js:69:7)
    at /Users/rogersmac/ML/authCenter/node_modules/co/index.js:54:5
    at new Promise (<anonymous>)
    at co (/Users/rogersmac/ML/authCenter/node_modules/co/index.js:50:10)
    at /Users/rogersmac/ML/authCenter/utils/controller/index.js:6:9
    at Layer.handle [as handle_request] (/Users/rogersmac/ML/authCenter/node_modules/express/lib/router/layer.js:95:5)
    at next (/Users/rogersmac/ML/authCenter/node_modules/express/lib/router/route.js:137:13)
    at Route.dispatch (/Users/rogersmac/ML/authCenter/node_modules/express/lib/router/route.js:112:3)
    at Layer.handle [as handle_request] (/Users/rogersmac/ML/authCenter/node_modules/express/lib/router/layer.js:95:5)
    at /Users/rogersmac/ML/authCenter/node_modules/express/lib/router/index.js:281:22
    at Function.process_params (/Users/rogersmac/ML/authCenter/node_modules/express/lib/router/index.js:335:12)
    at next (/Users/rogersmac/ML/authCenter/node_modules/express/lib/router/index.js:275:10)
    at Function.handle (/Users/rogersmac/ML/authCenter/node_modules/express/lib/router/index.js:174:3) name: 401 }
[2019-02-23 11:36:04.982] [ERROR] console - { 401: authorization failed
    at fail (/Users/rogersmac/ML/authCenter/utils/controller/index.js:47:19)
    at /Users/rogersmac/ML/authCenter/apps/auth/app.js:62:13
    at Generator.next (<anonymous>)
    at onFulfilled (/Users/rogersmac/ML/authCenter/node_modules/co/index.js:65:19)
    at /Users/rogersmac/ML/authCenter/node_modules/co/index.js:54:5
    at new Promise (<anonymous>)
    at co (/Users/rogersmac/ML/authCenter/node_modules/co/index.js:50:10)
    at toPromise (/Users/rogersmac/ML/authCenter/node_modules/co/index.js:118:63)
    at next (/Users/rogersmac/ML/authCenter/node_modules/co/index.js:99:29)
    at onFulfilled (/Users/rogersmac/ML/authCenter/node_modules/co/index.js:69:7)
    at /Users/rogersmac/ML/authCenter/node_modules/co/index.js:54:5
    at new Promise (<anonymous>)
    at co (/Users/rogersmac/ML/authCenter/node_modules/co/index.js:50:10)
    at /Users/rogersmac/ML/authCenter/utils/controller/index.js:6:9
    at Layer.handle [as handle_request] (/Users/rogersmac/ML/authCenter/node_modules/express/lib/router/layer.js:95:5)
    at next (/Users/rogersmac/ML/authCenter/node_modules/express/lib/router/route.js:137:13)
    at Route.dispatch (/Users/rogersmac/ML/authCenter/node_modules/express/lib/router/route.js:112:3)
    at Layer.handle [as handle_request] (/Users/rogersmac/ML/authCenter/node_modules/express/lib/router/layer.js:95:5)
    at /Users/rogersmac/ML/authCenter/node_modules/express/lib/router/index.js:281:22
    at Function.process_params (/Users/rogersmac/ML/authCenter/node_modules/express/lib/router/index.js:335:12)
    at next (/Users/rogersmac/ML/authCenter/node_modules/express/lib/router/index.js:275:10)
    at Function.handle (/Users/rogersmac/ML/authCenter/node_modules/express/lib/router/index.js:174:3) name: 401 } '401: authorization failed\n    at fail (/Users/rogersmac/ML/authCenter/utils/controller/index.js:47:19)\n    at /Users/rogersmac/ML/authCenter/apps/auth/app.js:62:13\n    at Generator.next (<anonymous>)\n    at onFulfilled (/Users/rogersmac/ML/authCenter/node_modules/co/index.js:65:19)\n    at /Users/rogersmac/ML/authCenter/node_modules/co/index.js:54:5\n    at new Promise (<anonymous>)\n    at co (/Users/rogersmac/ML/authCenter/node_modules/co/index.js:50:10)\n    at toPromise (/Users/rogersmac/ML/authCenter/node_modules/co/index.js:118:63)\n    at next (/Users/rogersmac/ML/authCenter/node_modules/co/index.js:99:29)\n    at onFulfilled (/Users/rogersmac/ML/authCenter/node_modules/co/index.js:69:7)\n    at /Users/rogersmac/ML/authCenter/node_modules/co/index.js:54:5\n    at new Promise (<anonymous>)\n    at co (/Users/rogersmac/ML/authCenter/node_modules/co/index.js:50:10)\n    at /Users/rogersmac/ML/authCenter/utils/controller/index.js:6:9\n    at Layer.handle [as handle_request] (/Users/rogersmac/ML/authCenter/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/rogersmac/ML/authCenter/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/Users/rogersmac/ML/authCenter/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/Users/rogersmac/ML/authCenter/node_modules/express/lib/router/layer.js:95:5)\n    at /Users/rogersmac/ML/authCenter/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/Users/rogersmac/ML/authCenter/node_modules/express/lib/router/index.js:335:12)\n    at next (/Users/rogersmac/ML/authCenter/node_modules/express/lib/router/index.js:275:10)\n    at Function.handle (/Users/rogersmac/ML/authCenter/node_modules/express/lib/router/index.js:174:3)'
[2019-02-23 11:36:18.487] [INFO] console - ******** getResFn error ******** 
[2019-02-23 11:36:18.489] [INFO] console - { 401: authorization failed
    at fail (/Users/rogersmac/ML/authCenter/utils/controller/index.js:47:19)
    at /Users/rogersmac/ML/authCenter/apps/auth/app.js:62:13
    at Generator.next (<anonymous>)
    at onFulfilled (/Users/rogersmac/ML/authCenter/node_modules/co/index.js:65:19)
    at /Users/rogersmac/ML/authCenter/node_modules/co/index.js:54:5
    at new Promise (<anonymous>)
    at co (/Users/rogersmac/ML/authCenter/node_modules/co/index.js:50:10)
    at toPromise (/Users/rogersmac/ML/authCenter/node_modules/co/index.js:118:63)
    at next (/Users/rogersmac/ML/authCenter/node_modules/co/index.js:99:29)
    at onFulfilled (/Users/rogersmac/ML/authCenter/node_modules/co/index.js:69:7)
    at /Users/rogersmac/ML/authCenter/node_modules/co/index.js:54:5
    at new Promise (<anonymous>)
    at co (/Users/rogersmac/ML/authCenter/node_modules/co/index.js:50:10)
    at /Users/rogersmac/ML/authCenter/utils/controller/index.js:6:9
    at Layer.handle [as handle_request] (/Users/rogersmac/ML/authCenter/node_modules/express/lib/router/layer.js:95:5)
    at next (/Users/rogersmac/ML/authCenter/node_modules/express/lib/router/route.js:137:13)
    at Route.dispatch (/Users/rogersmac/ML/authCenter/node_modules/express/lib/router/route.js:112:3)
    at Layer.handle [as handle_request] (/Users/rogersmac/ML/authCenter/node_modules/express/lib/router/layer.js:95:5)
    at /Users/rogersmac/ML/authCenter/node_modules/express/lib/router/index.js:281:22
    at Function.process_params (/Users/rogersmac/ML/authCenter/node_modules/express/lib/router/index.js:335:12)
    at next (/Users/rogersmac/ML/authCenter/node_modules/express/lib/router/index.js:275:10)
    at Function.handle (/Users/rogersmac/ML/authCenter/node_modules/express/lib/router/index.js:174:3) name: 401 }
[2019-02-23 11:36:18.493] [ERROR] console - { 401: authorization failed
    at fail (/Users/rogersmac/ML/authCenter/utils/controller/index.js:47:19)
    at /Users/rogersmac/ML/authCenter/apps/auth/app.js:62:13
    at Generator.next (<anonymous>)
    at onFulfilled (/Users/rogersmac/ML/authCenter/node_modules/co/index.js:65:19)
    at /Users/rogersmac/ML/authCenter/node_modules/co/index.js:54:5
    at new Promise (<anonymous>)
    at co (/Users/rogersmac/ML/authCenter/node_modules/co/index.js:50:10)
    at toPromise (/Users/rogersmac/ML/authCenter/node_modules/co/index.js:118:63)
    at next (/Users/rogersmac/ML/authCenter/node_modules/co/index.js:99:29)
    at onFulfilled (/Users/rogersmac/ML/authCenter/node_modules/co/index.js:69:7)
    at /Users/rogersmac/ML/authCenter/node_modules/co/index.js:54:5
    at new Promise (<anonymous>)
    at co (/Users/rogersmac/ML/authCenter/node_modules/co/index.js:50:10)
    at /Users/rogersmac/ML/authCenter/utils/controller/index.js:6:9
    at Layer.handle [as handle_request] (/Users/rogersmac/ML/authCenter/node_modules/express/lib/router/layer.js:95:5)
    at next (/Users/rogersmac/ML/authCenter/node_modules/express/lib/router/route.js:137:13)
    at Route.dispatch (/Users/rogersmac/ML/authCenter/node_modules/express/lib/router/route.js:112:3)
    at Layer.handle [as handle_request] (/Users/rogersmac/ML/authCenter/node_modules/express/lib/router/layer.js:95:5)
    at /Users/rogersmac/ML/authCenter/node_modules/express/lib/router/index.js:281:22
    at Function.process_params (/Users/rogersmac/ML/authCenter/node_modules/express/lib/router/index.js:335:12)
    at next (/Users/rogersmac/ML/authCenter/node_modules/express/lib/router/index.js:275:10)
    at Function.handle (/Users/rogersmac/ML/authCenter/node_modules/express/lib/router/index.js:174:3) name: 401 } '401: authorization failed\n    at fail (/Users/rogersmac/ML/authCenter/utils/controller/index.js:47:19)\n    at /Users/rogersmac/ML/authCenter/apps/auth/app.js:62:13\n    at Generator.next (<anonymous>)\n    at onFulfilled (/Users/rogersmac/ML/authCenter/node_modules/co/index.js:65:19)\n    at /Users/rogersmac/ML/authCenter/node_modules/co/index.js:54:5\n    at new Promise (<anonymous>)\n    at co (/Users/rogersmac/ML/authCenter/node_modules/co/index.js:50:10)\n    at toPromise (/Users/rogersmac/ML/authCenter/node_modules/co/index.js:118:63)\n    at next (/Users/rogersmac/ML/authCenter/node_modules/co/index.js:99:29)\n    at onFulfilled (/Users/rogersmac/ML/authCenter/node_modules/co/index.js:69:7)\n    at /Users/rogersmac/ML/authCenter/node_modules/co/index.js:54:5\n    at new Promise (<anonymous>)\n    at co (/Users/rogersmac/ML/authCenter/node_modules/co/index.js:50:10)\n    at /Users/rogersmac/ML/authCenter/utils/controller/index.js:6:9\n    at Layer.handle [as handle_request] (/Users/rogersmac/ML/authCenter/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/rogersmac/ML/authCenter/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/Users/rogersmac/ML/authCenter/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/Users/rogersmac/ML/authCenter/node_modules/express/lib/router/layer.js:95:5)\n    at /Users/rogersmac/ML/authCenter/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/Users/rogersmac/ML/authCenter/node_modules/express/lib/router/index.js:335:12)\n    at next (/Users/rogersmac/ML/authCenter/node_modules/express/lib/router/index.js:275:10)\n    at Function.handle (/Users/rogersmac/ML/authCenter/node_modules/express/lib/router/index.js:174:3)'
[2019-02-23 11:46:21.700] [INFO] console - Express server listening on :8999, in development mode
[2019-02-23 11:46:21.739] [INFO] console - mongo连接成功
[2019-02-23 11:46:21.741] [INFO] console - mongo连接打开
